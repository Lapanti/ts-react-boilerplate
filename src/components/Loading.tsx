import * as React from 'react';

const Loading: React.StatelessComponent<undefined> = () => (
    <i className="loading">
        Loading
    </i>
);

export default Loading;
