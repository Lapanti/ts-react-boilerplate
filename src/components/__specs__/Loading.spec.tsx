import * as React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';

describe('Loading', () => (
    it('should render correctly', () => (
        expect(shallow(<Loading />)).toMatchSnapshot()
    ))
));
