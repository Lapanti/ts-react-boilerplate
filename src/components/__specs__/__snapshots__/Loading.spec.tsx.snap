// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
<i
  className="loading"
>
  Loading
</i>
`;
